contracts/utilityToken/ERC20Token.sol

```diff
@@ -0,0 +1,256 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "../ERC20I.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+ * @title ERC20Token contract.
+ *
+ * @notice ERC20Token implements ERC20I.
+ */
+contract ERC20Token is ERC20I {
+    /* Usings */
+
+    using SafeMath for uint256;
+
+    /** Name of the token. */
+    string internal tokenName;
+
+    /** Symbol of the token. */
+    string internal tokenSymbol;
+
+    /** Decimals used by the token. */
+    uint8 internal tokenDecimals;
+
+    /** Total supply of the token. */
+    uint256 internal totalTokenSupply;
+
+    /** Stores the token balance of the accounts. */
```

Comment:
"Stores the token balance of the owners."

Reply:
Updated